libr/bin/format/wasm/wasm.c

```diff
@@ -7,51 +7,45 @@
 
 #include "wasm.h"
 
+typedef size_t (*ConsumeFcn)(const ut8 *p, const ut8 *max, ut32 *out_value);
```

Comment:
i'm sure we will not always have `ut32*` pointer there.

Reply:
Mmm I guess we'll take care of that when that's needed. Right now we don't need to complicate it. In any case, it will be easy to change if you need, for example, ut64 instead of ut32.

Reply:
ok.